docs/references/javascript/types/use-organization-params.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: useOrganizationParams
+description: Parameters for the `useOrganization()` hook.
+---
+
+# `useOrganizationParams`
+
+Parameters for the [`useOrganization()`](/docs/references/react/use-organization) hook.
+
+```typescript
+type useOrganizationParams = {
+  invitationList: {
+    // Pagination params
+    limit?: number;
+    offset?: number;
+    },
+  membershipRequests: true
+    | (commonPaginationParams & {
+        status?: OrganizationInvitationStatus; // e.g. "pending"
+      });
+  memberships: true
+    | (commonPaginationParams & {
+        role?: MembershipRole[]; // e.g. ["basic_member", "admin"]
+      });
+  domains: true
+    | (commonPaginationParams & {
+        enrollmentMode?: OrganizationEnrollmentMode; // e.g. "automatic_invitation"
```

Comment:
Shall we just enter the actual union type here ? People don't know what `OrganizationEnrollmentMode` as there is no other reference to the docs

Reply:
Don't you think it's better to only have `/types` for top level types ?

Reply:
Docs is getting a huge reorganization soon - so whatever feels more comfortable. Let's just make sure we are documenting anything we can - so if we want to a table on the same page for `CommonPaginationParams`, that's fine and let's make sure we do it for `OrganizationInvitationStatus`, `MembershipRole`, and OrganizationEnrollmentMode` too

Reply:
sounds good, i can help if needed, lmk